modules/plex.py

```diff
@@ -846,11 +846,12 @@ def myPlexSignin(self, username='', password=''):
 
             username = username or htpc.settings.get('plex_username', '')
             password = password or htpc.settings.get('plex_password', '')
+            credentials = f'{quote_plus(username)}:{quote_plus(password)}'
 
             if username and password:
                 self.logger.debug('Fetching auth token')
                 headers = {}
-                headers['Authorization'] = 'Basic %s' % base64.encodestring('%s:%s' % (username, password)).replace('\n', '')
+                headers['Authorization'] = 'Basic %s' % base64.encodebytes(credentials.encode()).decode().replace('\n', '')
```

Comment:
Little enhancement: https://docs.python.org/3/library/base64.html
- `encodebytes` is `b64encode` + added newlines after every 76 characters + end, which we do not need and actively remove here. So let's use `b64encode` and skip having the newlines added and manually removing them again.
```suggestion
                headers['Authorization'] = f'Basic {base64.b64encode(credentials.encode()).decode()}'
```